iphone/Classes/TiUIListView.m

```diff
@@ -528,9 +526,6 @@ - (void)updateSearchResults:(id)unused
         [self buildResultsForSearchText];
     }
     [_tableView reloadData];
-    if ([searchController isActive]) {
-        [[searchController searchResultsTableView] reloadData];
-    } 
```

Comment:
SearchViewController is using current tableView. And it do not have searchResultsTableView.